recipes/qtxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conans import ConanFile, tools
+
+
+class QtXlsxWriterConan(ConanFile):
+    name = "qtxlsxwriter"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dbzhang800/QtXlsxWriter"
+    description = ".xlsx file reader and writer for Qt5"
+    topics = ("qtxlsxwriter", "excel", "xlsx", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "CMakeDeps"
```

Comment:
🟨 For the conan team, can we start using this in CCI or is it still likely to break?

Reply:
This is a very experimental generator and I think we cannot start using it in ConanCenter yet